common-utils/src/main/java/eu/arrowhead/common/collector/HttpCollectorDriver.java

```diff
@@ -108,11 +117,11 @@ private ServiceModel acquireServiceFromSR(final String serviceDefinitionName, fi
 		}
 
 		final ServiceInstanceListResponseDTO response = httpService.sendRequest(uri, HttpMethod.POST, ServiceInstanceListResponseDTO.class, payload, null, headers);
-		return convertLookupResponse(response);
+		return convertLookupResponse(response, providerName);
 	}
 
 	//-------------------------------------------------------------------------------------------------
-	private ServiceModel acquireServiceFromOrchestrator(final String serviceDefinitionName, final String interfaceTemplateName) {
+	private ServiceModel acquireServiceFromOrchestrator(final String serviceDefinitionName, final String interfaceTemplateName, final String providerName) {
```

Comment:
we should rename this to acquireServiceFromOrchestration to show consistency (however I wouldn't put an extra Service in the name).